src/mtsoroyematatu.c

```diff
@@ -0,0 +1,785 @@
+/************************************************************************
+**
+** NAME:        mtsoroyematatu.c
+**
+** DESCRIPTION: Tsoro Yematatu
+**
+** AUTHOR:      Ken Zheng
+**              Justin Park
+**              Michael SP
+**              Jason Ding
+**
+** DATE:        2024-11-20
+**
+************************************************************************/
+
+#include "gamesman.h"
+
+CONST_STRING kAuthorName = "Ken Zheng, Justin Park, Michael SP, Jason Ding";
+CONST_STRING kGameName = "Tsoro Yematatu"; 
+CONST_STRING kDBName = "tsoroyematatu";
+
+/**
+ * @brief An upper bound on the number of reachable positions.
+ *
+ * @details The hash value of every reachable position must be less
+ * than `gNumberOfPositions`.
+ */
+
+
+POSITION gNumberOfPositions = 6561; // > 3^7
```

Comment:
Since it's here, update the comment to be 3^8 (since we added the turn encoding to the position).